fig.yml

```diff
@@ -0,0 +1,18 @@
+galera1:
+  build: .
+  command: --wsrep-cluster-address=gcomm://
```

Comment:
Yes, exactly. Galera needs one peer to be started as the instance that initializes the cluster. After other peers joined all of them are equal.